nntrainer/tensor/cl_operations/testing_rotarty_emb.cpp

```diff
@@ -0,0 +1,182 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2024 Yash Singh <yash.singh@samsung.com>
+ *
+ * @file	testing_rotary_emb.cpp
```

Comment:
if this file is for the testing.. then it might be better to be in unittest dir.

Reply:
Moved the file to unittest directory in the latest commit.